integration/dbutils/migrator.spec.ts

```diff
@@ -12,6 +12,14 @@ var dbShape : Shapes.DBShape = {
   tables: [{
     tableName: 'users',
     indices: ['userName', 'email']
+  },
+  {
+    tableName: 'userData',
+    indices: []
+  },
+  {
+    tableName: 'activation',
```

Comment:
And since my second paragraph isn't really a blocker at this point, I'll add #peer to it once the build completes ;)